src/utils/d3-decorators.js

```diff
@@ -1,19 +1,15 @@
-import {utils} from '../utils/utils';
-import {utilsDom} from '../utils/utils-dom';
-import {utilsDraw} from '../utils/utils-draw';
-import {default as _} from 'underscore';
+import {utils} from './utils';
+import {utilsDom} from './utils-dom';
+import {utilsDraw} from './utils-draw';
 import {default as d3} from 'd3';
-// TODO: Fix utilsDom export.
-var selectOrAppend = utilsDom.selectOrAppend;
-var classes = utilsDom.classes;
 
-var d3getComputedTextLength = _.memoize(
+var d3getComputedTextLength = () => utils.memoize(
```

Comment:
Because otherwise I get and error saying that utils is undefined. I guess it has to do with the way webpack import the dependencies but no idea of what the exact issue is